src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private string GetActivationCodeFromDb(string userEmail)
+        {
+            return "pttigkgkgkgkgk6666k6k6k6k6k6k6k1123ndncj";
+        }
+
+        private string GetNameFromDb(string userEmail)
+        {
+            return "Mike";
+        }
+
+        private static string GetBaseUrl()
+        {
+            return "Test.com";
+        }
+
+        private string GetActivationUrl(string userEmail)
+        {
+            string baseUrl = GetBaseUrl();
+            string activationCode = GetActivationCodeFromDb(userEmail);
+            string activationUrl = baseUrl + "/TestAppName/ActivateAccount/" + activationCode;
```

Comment:
TestAppName? what is that?

Reply:
Lol. I'll replace it with Dwit-Tech-App. Is that ok?

Reply:
No thats not okay